DemoApp/app/App.js

```diff
@@ -26,8 +26,20 @@ export default createBottomTabNavigator(
   }
 );
 
+let previousAppState = AppState.currentState;
+export const setPreviousAppState = (appState) => {
+  previousAppState = appState;
+  // Workaround to distinguish between notifications received in the foreground or while the app was in the background.
+  setTimeout(() => {
+    previousAppState = AppState.currentState;
+  }, 1000);
+};
+
 Push.setListener({
   onPushNotificationReceived(pushNotification) {
+    if (previousAppState !== 'active') {
```

Comment:
Question - I saw `Alert.alert` call in line 41 and line 71, will you get 2 alerts when receiving push in background?

Reply:
thanks for confirming, combining into one alert sounds good.